tests/integration/cli/restart_test.py

```diff
@@ -0,0 +1,71 @@
+import datetime
+import os
+import time
+
+import dateutil.parser
+from nose.tools import nottest
+
+from dusty.compiler.spec_assembler import get_all_repos
+from dusty.source import Repo
+from ...testcases import DustyIntegrationTestCase
+from ...fixtures import specs_fixture_with_depends
+
+class TestRestartCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestRestartCLI, self).setUp()
+        specs_fixture_with_depends()
+        for repo in get_all_repos(include_specs_repo=False):
+            self._set_up_fake_local_repo(path=repo.remote_path)
```

Comment:
might want to rename this if it's not private anymore.. i guess it is private to the class not sure what convention is on that